### PR TITLE
[SHIRO-866] 1.9.0 update and announcement

### PR DESCRIPTION
**Pull request by bmarwell**
fixes [SHIRO-866](https://issues.apache.org/jira/browse/SHIRO-866).

@fpapon the date is set to this friday (2022-02-18).
We will probably need to change this, alone for the 72h vote cycle.

### CODE REVIEW

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -11,13 +30,14 @@ Git will automatically convert line endings for you.
...
-* If the new `latestRelease` is a minor release, update the version number of the `releases` object at the end of the file. +
-_Rationale:_ Every minor release should only be represented with the latest bugfix release.
 * Change `latestRelease: "..."` to the new version.
 * Update the `versionInfo` object accordingly. +
 You can just add a new release, there is no need to remove old releases.
+* Update the `oldReleases`/`releases` variable like so:
+** If the new `latestRelease` is a new minor release (e.g. `1.n.0`, where `n` is the new minor release version), prepend the old version to the `oldReleases` array. +
+Also add it to the 'releases' object at the end of the file as `1nx`.
+** If the new `latestRelease` is a bugfix release (e.g. `1.n.x`), update the version number of the `releases` object at the end of the file. +
+_Rationale:_ Every minor release should only be represented with the latest bugfix release.
```
Comment (bdemers) [inline CONTRIBUTING.adoc:40]:
```suggestion
* Update the `oldReleases`/`releases` variable:
** If the new `latestRelease` is a new minor release (e.g. `1.n.0`, where `n` is the new minor release version), prepend the old version to the `oldReleases` array. +
Also, add it to the 'releases' object at the end of the file as `1nx`.
** If the new `latestRelease` is a bugfix release (e.g. `1.n.x`), update the version number of the `releases` object at the end of the file. +
_Rationale:_ Every minor release should only be represented with the latest bugfix release.
```

**Review by bdemers — APPROVED** [review]
Looks great! just minor nits.

**Review by fpapon — APPROVED** [review]

**Comment by fpapon** [comment]
@bmarwell release date has been updated

**Review by bmarwell — COMMENTED** [review]
LGTM

File: src/site/content/blog/2022/03/22/apache-shiro-190-released.adoc [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+////
+
+= Apache Shiro 1.9.0 Released
+Benjamin Marwell
+:jbake-date: 2022-02-22 21:39:37
```
Comment (bmarwell) [inline src/site/content/blog/2022/03/22/apache-shiro-190-released.adoc:22]:
```suggestion
:jbake-date: 2022-03-22 21:39:37
```

**Pull request merged**